ml-conversational-analytic-tool/githubDataExtraction.py

```diff
@@ -157,23 +211,21 @@ def getPullFeatures(self, pull):
     parser.add_argument('-R', '--repo', help='Name of repo.')
     parser.add_argument('--reactions', action='store_true', default=False, help='Flag to extract reactions')
     parser.add_argument('-n', "--name", help='Output file name. If not specified, the name is constructed like this: '
-            '<organization>_<repo>.csv')
+                                             '<organization>_<repo>.csv')
 
     args = parser.parse_args()
     ACCESS_TOKEN = os.environ["GITACCESS"]  # Access Github token from environment for security purposes
-    extractor = GithubDataExtractor(ACCESS_TOKEN)  # Create object
+    extractor = GithubDataExtractor(ACCESS_TOKEN, args.organization)
 
     if args.repo is None:
         # Extract data for all repositories in organization
-        repos = extractor.g_ses.get_organization(args.organization).get_repos()
+        repos = extractor.get_all_repos()
         for repo in repos:
-            extractor.openRepo(args.organization, repo.name)
-            df = extractor.getAllPulls(args.reactions)
-            file_name = utils.construct_file_name(None, args.organization, repo.name)
+            df = extractor.get_all_pull_requests(repo, args.reactions)
+            file_name = utils.construct_file_name(None, args.organization, repo)
             utils.export_to_cvs(df, file_name)
     else:
         # Extract data for an individual repository
-        extractor.openRepo(args.organization, args.repo)
-        df = extractor.getAllPulls(args.reactions)
+        df = extractor.get_all_pull_requests(args.repo, args.reactions)
```

Comment:
These three lines are repeated for each repository. Could go into a separate function I think.

Reply:
refactored to create a separate function